modules/opm-cli-ref-generate.adoc

```diff
@@ -63,12 +57,15 @@ $ opm generate dockerfile <dcRootDir> [flags]
 |Flag |Description
 
 |`-i,` `--binary-image` (string)
-|Image in which to build catalog. The default value is `quay.io/operator-framework/opm:latest`.
+|Base image with which to build the catalog. The default value is the upstream `quay.io/operator-framework/opm:latest` image.
```

Comment:
As per https://github.com/openshift/openshift-docs/pull/59181/files#r1187638829 we shouldn't unilaterally update the docs' display of the command output without updating the command output in the binary.  If there's already a tracking issue for this change, would you include it in the PR description please?

Reply:
Same re: https://github.com/openshift/openshift-docs/pull/59181#discussion_r1188727603.